src/libexpr/parser.y

```diff
@@ -365,6 +365,15 @@ string_parts_interpolated
 path_start
   : PATH {
     std::string_view literal({$1.p, $1.l});
+
+    /* check for short path literals */
+    if (state->settings.warnShortPathLiterals && literal.front() != '/' && literal.front() != '.') {
+        logWarning({
+            .msg = HintFmt("Short path literal '%s' is deprecated. Use './%s' instead.", literal, literal),
```

Comment:
I'm not sure we should phrase it as "policy"; perhaps a more factual message could be:

```suggestion
            .msg = HintFmt("relative path literal '%s' should be prefixed with '.' for clarity: './%s'. (" ANSI_BOLD "warn-short-path-literals" ANSI_NORMAL " = true)", literal, literal),
```